signatures.yaml

```diff
@@ -265,13 +273,6 @@
 - Postgresql:
     - URL: (?i)(?:pgsql:|postgres:|postgresql:)//[\S]{1,256}:[\S]{1,256}@[-.%\w\/:]+\.[\S]+
 
-- GitHub:
```

Comment:
Sorry for asking but is this intentional? I suppose so seeing you also removed the GitHub test case below. In any case shouldn't we still have tests that cover github's use cases?

Reply:
yes, the problem was that due to a previous bug, all the tokens from github were hardcoded, not picked from the yaml file. When i removed that hardcoding and started to use the regexes from the yaml, i needed to modify tests, and also remove a useless one that was testing the hardcoded behaviour